src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/jclouds/JCloudsComputeServiceBuilder.java

```diff
@@ -67,16 +97,24 @@ public ComputeService buildComputeServiceFromInfrastructure(Infrastructure infra
      * 
      * @return Properties object with the timeout policy.
      */
-    private Properties getTimeoutPolicy() {
+    private Properties getOverrides(Infrastructure infrastructure) {
```

Comment:
Maybe getDefinedProperties will be clearer than getOverrides

Reply:
Done, thanks